### ISSUE TITLE
kubes version to be used

### ISSUE THREAD

**Issue by goern** [LalatenduMohanty/centos7-container-app-vagrant-box#14]
RHEL is going to release 0.15.0-0.3.git0ea87e4 of kubernetes tomorrow, it will support v1beta3 kubernetes API, we should make a decision if we include kubernetes-0.16.2-2.el7 into the vagrant box.

**Comment by kbsingh** [comment]
At the moment, the builds consume the downstream of whatever is released via the RHEL release process. We opted for going down the conservative route of just consuming the default package set. i.e once the Kuberneties updates are released, we should have access to them in existing box's with a 'yum update' and we can potentially create a new box with the updates pre-applied.

I've also got an issue open around doing a second box, that tracks the upstream versions of code a lot closer, and perhaps uses a nightly docker / kuberneties build instead of the downstream one. Would having this second box ( so folks can opt into the bleeding edge ) be a better overall solution than cherry picking versions into the default dev box ? ref: https://github.com/LalatenduMohanty/centos7-container-app-vagrant-box/issues/17
